handout/routing/redirects.md

```diff
@@ -11,6 +11,7 @@ export const routes: Routes = [
 ];
 ```
 
-This tells the router to redirect to component-one when matching the empty path ('').
+The `pathMatch` property, which is required, tells the router how it should match the URL provided in order to redirect to the specified route. 
+Since `pathMatch: full` is provided, the router will redirect to `component-one` if the entire URL matches the empty path ('').
```

Comment:
Are these two sentences different paragraphs or the same one? If two different paragraphs we need to add a blank line. If the same paragraph you should remove the break line